### PR TITLE
Typescript migration of selection control stories

### PR DESCRIPTION
**Pull request by wenche**

### CODE REVIEW

File: apps/storybook-react/stories/SelectionControls.Checkbox.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+  const { register, handleSubmit, watch, errors } = useForm<FormData>()
+  const [isSubmitted, updateIsSubmitted] = useState(false)
+  const [formData, updateFormData] = useState<FormData>(null)
+
+  const onSubmit = (data: FormData) => {
+    updateFormData(data)
+    updateIsSubmitted(true)
+    action('onSubmit')(data)
+  }
+  console.log(watch('example'))
```
Comment (pomfrida) [inline apps/storybook-react/stories/SelectionControls.Checkbox.stories.tsx:118]: 🔥  Remove logs

**Review by pomfrida — CHANGES_REQUESTED** [review]
Remove `consol.log`, looks good otherwise 👍

File: apps/storybook-react/stories/SelectionControls.Checkbox.stories.tsx (hunk 1 above)
Comment (wenche) [reply apps/storybook-react/stories/SelectionControls.Checkbox.stories.tsx:118]: It's from the react-hook-library and is supposed to show what's going on inside it. Documented at their page. :)

Comment (pomfrida) [reply apps/storybook-react/stories/SelectionControls.Checkbox.stories.tsx:118]: Cool!

**Review by pomfrida — APPROVED** [review]
LGTM👍

**Pull request merged**